docs/testing/debug-add-ins-overview.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Debug Office Add-ins
+description: 'Find the Office Add-in debugging guidance for your development environment'
+ms.date: 12/02/2021
+ms.localizationpriority: high
+---
+
+# Overview of debugging Office Add-ins
+
+Debugging Office Add-ins is essentially the same as debugging any web application; but there isn't a single set of tools that all add-in developers can use. This is because add-ins can be developed on different operating systems and can run cross-platform. This article will help you find the detailed debugging guidance for your development environment.
+
+> [!NOTE]
+> This article is concerned with debugging in the narrow sense of setting breakpoints and stepping through code. For guidance on testing and troubleshooting, start with [Test Office Add-ins](test-debug-office-add-ins.md) and [Troubleshoot development errors with Office Add-ins](troubleshoot-development-errors.md).
+>
+> This article uses "your development computer" and "your development environment" to refer to the environment on which you're debugging. This almost always *is* your development computer. Although you should *test* your add-in on all the platforms that you want to support, you'll only very rarely need to *debug* on an environment different from your development computer.  
```

Comment:
What confusion is this note about development computer/environment trying to solve? What computer would people think we're talking about?

Reply:
What I'm trying to get a across with the note is that normally you don't have to debug on more than one platform. But once in a rare while you'll have to debug on an environment that is different from the one your dev machine is using. So, I want to use "your development computer" instead of "the environment on which you're debugging" because the latter gives the false impression that you have to debug on a lot of environments other than the one on your dev machine. The note is trying to explain this editorial decision. I'll rewrite.